python/setup_ocnice_config.py

```diff
@@ -44,9 +44,9 @@ def check_env(envVarName):
 
 add_config_option(config, 'input', 'baseDirectory', baseDir)
 add_config_option(config, 'input', 'runSubdirectory', 'run')
-add_config_option(config, 'input', 'oceanNamelistFileName', 'run/mpas-o_in')
+add_config_option(config, 'input', 'oceanNamelistFileName', 'mpas-o_in')
 add_config_option(config, 'input', 'oceanStreamsFileName', 'run/streams.ocean')
-add_config_option(config, 'input', 'seaIceNamelistFileName', 'run/mpas-cice_in')
+add_config_option(config, 'input', 'seaIceNamelistFileName', 'mpas-cice_in')
```

Comment:
These changes should be reverted and instead the bug fix https://github.com/MPAS-Dev/MPAS-Analysis/pull/210 should be merged into MPAS-Analysis.